hikari/api/event_manager.py

```diff
@@ -327,7 +327,7 @@ def get_listeners(
         /,
         *,
         polymorphic: bool = True,
-    ) -> typing.Collection[CallbackT[EventT_co]]:
+    ) -> typing.Sequence[CallbackT[EventT_co]]:
```

Comment:
There's no guarantee this will be sorted as an ordered dataset for all impls though